packages/codegen-sdk/src/print-connection.ts

```diff
@@ -58,15 +61,15 @@ function printConnectionVariables(): string {
 function printFetchType(): string {
   return printLines([
     printComment(["Fetch return type wrapped in a promise"]),
-    `export type ${Sdk.FETCH_TYPE}<${Sdk.RESPONSE_TYPE}> = Promise<${Sdk.RESPONSE_TYPE} | undefined>`,
+    `export type ${Sdk.FETCH_TYPE}<${Sdk.RESPONSE_TYPE}> = Promise<${Sdk.RESPONSE_TYPE}>`,
```

Comment:
The `LinearFetch` response type is now just a `Promise`, nothing else. This means we need to add `| undefined` to a few bits of the codegen where fields are not non-nullable.